cognite_toolkit/_cdf_tk/loaders/_resource_loaders/location_loaders.py

```diff
@@ -56,6 +59,22 @@ class LocationFilterLoader(
 
     subfilter_names = ("assets", "events", "files", "timeseries", "sequences")
 
+    def __init__(self, client: ToolkitClient, build_dir: Path | None, console: Console | None = None) -> None:
+        self.client = client
```

Comment:
missing a super call
```suggestion
super().__init__(client, build_dir, console)
```